mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown

```diff
@@ -432,3 +432,9 @@ resource "google_project_iam_binding" "gcs-bucket-writer" {
 ### resource `google_cloudiot_registry_iam_*` is now removed
 
 ### datasource `google_cloudiot_registry_iam_policy` is now removed
+
+## Resource: `google_service_networking_connection`
+
+### `Create` method is used to create the resource
+
+Previously, Patch instead of Create is used to create the resource, because an error message is returned when creating multiple SQL instances after creating `google_service_networking_connection` with Create method. See more details in https://issuetracker.google.com/issues/131908322. Now the issue has been fixed. The initial workaround of using Patch is no longer needed. Create method is used to create the resource.
```

Comment:
```suggestion
Previously, Patch instead of Create is used to create the resource, because an error message is returned when creating multiple SQL instances after creating `google_service_networking_connection` with Create method. The initial workaround of using Patch is no longer needed.
```
- I don't think we need to include the issue tracker, it wasn't particularly helpful for me understanding the issue.
- When you say "The initial workaround of using Patch is no longer needed" are you telling users to do something, or is it already taken care of? 
- Are there any other specific steps the user needs to take when upgrading?
- Also, can we be more specific about Patch and Create? Like "Patch API endpoint" or something?

Reply:
Thanks for the suggestion.

* When you say "The initial workaround of using Patch is no longer needed" are you telling users to do something, or is it already taken care of?
  *  It is already taken care by using Create endpoint
* Are there any other specific steps the user needs to take when upgrading?
  * The user does not need to take any other steps.

Reply:
Thanks for the explanations!